doc/releases/migration-guide-3.5.rst

```diff
@@ -289,6 +289,12 @@ Required changes
   instead use the new driver Kconfig
   :kconfig:option:`CONFIG_DMA_MCUX_LPC_NUMBER_OF_CHANNELS_ALLOCATED`.
 
+* On all boards that use the LPC55XXX series SOC. The default core clock frequencies have been
+  updated to their fastest clock values. For most of the devices, this requires the use of a PLL.
```

Comment:
This is not true for `LPC55S36`. https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/arm/lpcxpresso55s36/doc/index.rst#system-clock

Reply:
https://github.com/zephyrproject-rtos/zephyr/blob/main/soc/arm/nxp_lpc/lpc55xxx/soc.c
https://docs.zephyrproject.org/latest/kconfig.html#CONFIG_INIT_PLL1

To me it looks like it is true for LPC55S36, the board documentation page is wrong, the default clock generator is the XTAL32K multiplied using PLL1 for all LPC55XXX now with this change, except for LPC55S06

Reply:
We need to update the board documentation to indicate we are running at 144MHz and not 150MHz. We should document the reasons for picking this frequency was to satisfy the CAN requirements. This should reflect in the Kconfig documentation for PLL1 as well. 
Also when updating this document, we should not be refer to this change as the fastest clock. Instead I would use the values and platforms impacted.